jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -753,6 +754,16 @@ function configure_jbpm_cache() {
         </cache-container>\n<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#g' ${CONFIG_FILE}
 }
 
+function configure_jbpm_cache_modules() {
+    sed -i 's#<system export="true">#<module name="org.infinispan" services="export"/><module name="org.jgroups"/>\n<system export="true">#g' ${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/jboss-deployment-structure.xml
```

Comment:
almost ready.
the output of this sed would be:

```
      <!-- required jdk dependencies for batik/svg transformation -->
      <module name="org.infinispan" services="export"/><module name="org.jgroups"/>
<system export="true">

```

can you please rework this to make the command output something like:

```
    
      <module name="org.infinispan" services="export"/>
      <module name="org.jgroups"/>
      <!-- required jdk dependencies for batik/svg transformation -->
      <system export="true">
            ...
```

Reply:
ok, I'll change to have on two lines